h2o-core/src/main/java/water/rapids/Env.java

```diff
@@ -51,6 +52,9 @@
  */
 public class Env extends Iced {
 
+  // for DEVELOPMENT/TESTING only
```

Comment:
Tiny tiny knit: Javadoc would show this description in IDEs as well :wink:

Reply:
thank you @Pscheidl !